app/checkresult/handler/check_result.py

```diff
@@ -2,7 +2,7 @@
 
 
 def lambda_handler(event, context):
-    client = Client(progress=False, wait_until_complete=False)
+    client = Client(progress=False, wait_until_complete=False, delete=False)
```

Comment:
What does this do?

Reply:
The `delete=False` flag makes sure that temporary files aren't removed and stored in /tmp, mostly for debugging purposes

Reply:
delete=False lets the extracted data file on their server remain after this call, so we can fetch it later. Otherwise we can't get it.